apps/core/components/userContext/UserContext.js

```diff
@@ -81,14 +108,19 @@ export default class UserContextProvider extends React.Component<Props, State> {
 
   firebaseListener: () => void;
 
-  mapUserData = (authUser: Object) => {
-    return {
-      userEmail: authUser?.email,
-      userDisplayName: authUser?.displayName,
-      profileImage: authUser?.photoURL,
-    };
+  getPhoneNumberFromFirebase = async (userId: string) => {
+    const userRef = firebase.database().ref(`users/${userId}/phoneNumber`);
+    const phoneNumbersnapshot = await userRef.once('value');
```

Comment:
`const phoneNumberSnapshot` with  capital Snapshot. Also, `userRef` is not really the user reference but the `phoneNumberRef`. We can also get the user reference and then `const phoneNumbersnapshot = await userRef.child('phoneNumber').once('value');` like we do on the `setUserPhoneNumber`, but I am perfectly okay with that 🚀.

Reply:
done